cdap-ui/app/cdap/components/PipelineScheduler/PipelineScheduler.scss

```diff
@@ -14,201 +14,161 @@
  * the License.
  */
 @import '../../styles/variables.scss';
+@import "../../styles/mixins.scss";
 
 $text-color: $grey-01;
 $border-color: $grey-05;
 $modeless-bg-color: $grey-06;
 $modeless-header-bg-color: $grey-08;
 $modeless-width: 730px;
 $modeless-height: 455px;
+$top-panel-height: 55px;
+$right-position: -330px;
 $regular-font: 14px;
 $secondary-btn-color: $brand-primary;
 
-body.theme-cdap {
-  &.state-hydrator {
-    .pipeline-scheduler-content {
-      background: $modeless-bg-color;
-      position: fixed;
-      top: 95px;
-      overflow: auto;
-      right: 15px;
-      z-index: 999;
-      box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.5);
-      color: $text-color;
-      background-color: white;
-      width: $modeless-width;
-      height: $modeless-height;
-
-      .pipeline-scheduler-header {
-        background-color: $modeless-header-bg-color;
-        height: 60px;
-        padding: 0;
-        border-radius: 4px 4px 0 0;
-        display: flex;
-        align-items: center;
-        justify-content: space-between;
-
-        .modeless-title {
-          color: $text-color;
-          padding-left: 15px;
-          max-width: 80%;
-          text-overflow: ellipsis;
-          white-space: nowrap;
-          overflow: hidden;
-          font-weight: 500;
-          margin: 0 auto 0 0;
-          font-size: 16px;
-          line-height: 20px;
-        }
+@include modeless();
```

Comment:
We should not do this. This will add `.modeless-container` on a global level and right now it is added in two places. So this will result in `.modeless-container` gets added twice.

![screen shot 2018-02-14 at 4 54 43 pm](https://user-images.githubusercontent.com/1452845/36235843-c7536b5e-11a7-11e8-8711-0c6559026eb0.png)